worker/test/src/RTC/RTCP/TestXr.cpp

```diff
@@ -0,0 +1,147 @@
+#include "common.hpp"
+#include "catch.hpp"
+#include "RTC/RTCP/XrDelaySinceLastRr.hpp"
+#include "RTC/RTCP/XrReceiverReferenceTime.hpp"
+#include <iostream>
+
+using namespace RTC::RTCP;
+
+SCENARIO("RTCP XrDelaySinceLastRt parsing", "[parser][rtcp][xr-dlrr]")
+{
+	SECTION("create RRT")
+	{
+		// Create local report and check content.
+		auto* report1 = new ReceiverReferenceTime();
+
+		report1->SetNtpSec(11111111);
+		report1->SetNtpFrac(22222222);
+
+		REQUIRE(report1->GetType() == ExtendedReportBlock::Type::RRT);
+		REQUIRE(report1->GetNtpSec() == 11111111);
+		REQUIRE(report1->GetNtpFrac() == 22222222);
+
+		// Serialize the report into an external buffer.
+		uint8_t bufferReport1[256]{ 0 };
+
+		report1->Serialize(bufferReport1);
+
+		// Create a new report out of the external buffer.
+		auto report2 = ReceiverReferenceTime::Parse(bufferReport1, report1->GetSize());
+
+		REQUIRE(report1->GetType() == report2->GetType());
+
+		REQUIRE(report1->GetNtpSec() == report2->GetNtpSec());
+		REQUIRE(report1->GetNtpFrac() == report2->GetNtpFrac());
+
+		// Create a local packet.
+		std::unique_ptr<ExtendedReportPacket> packet1(new ExtendedReportPacket());
+
+		packet1->SetSsrc(2222);
+		packet1->AddReport(report1);
+		packet1->AddReport(report2);
+
+		REQUIRE(packet1->GetType() == Type::XR);
+		REQUIRE(packet1->GetCount() == 0);
+		REQUIRE(packet1->GetSsrc() == 2222);
+
+		/* Total size:
```

Comment:
Do we really need this kind of `/* */` comments here? Can't we just use `//` lines as usual?